assets/prototype/domain/eventEditor/datesList/AddNewDateModal.js

```diff
@@ -0,0 +1,85 @@
+import classNames from "classnames";
+import { Classes, Overlay } from "@blueprintjs/core/lib/esm";
+// import { AppToaster } from "../EventEditor";
+import { Form, Field } from "react-final-form";
+import useCreateDateMutation from "../containers/mutations/useCreateDateMutation";
+
+const AddNewDateModal = ({ handleClose, isOpen }) => {
+	const [createDate, { data }] = useCreateDateMutation();
+
+	const onSubmit = async ({ name }) => {
+		createDate({ variables: { input: { name } } });
+	};
+
+	const overlayProps = {
+		autoFocus: true,
+		canEscapeKeyClose: true,
+		canOutsideClickClose: true,
+		enforceFocus: true,
+		hasBackdrop: true,
+		usePortal: true,
+		useTallContent: false
+	};
+
+	const OVERLAY_EXAMPLE_CLASS = "docs-overlay-example-transition";
+	const OVERLAY_TALL_CLASS = "docs-overlay-example-tall";
+
+	const classes = classNames(
+		Classes.CARD,
+		Classes.ELEVATION_4,
+		OVERLAY_EXAMPLE_CLASS,
+		OVERLAY_TALL_CLASS
+	);
+
+	const overlayStyle = {
+		top: "0",
+		left: "calc(50vw - 200px)",
+		margin: "10vh 0",
+		width: "400px"
+	};
+
+	return (
+		<Overlay
+			{...overlayProps}
+			className={Classes.OVERLAY_SCROLL_CONTAINER}
+			onClose={handleClose}
+			isOpen={isOpen}
+		>
+			<div className={classes} style={overlayStyle}>
+				<Form
+					onSubmit={onSubmit}
+					initialValues={{ stooge: "larry", employed: false }}
+					render={({ handleSubmit, form, submitting, pristine, values }) => (
+						<form onSubmit={handleSubmit}>
+							<div>
+								<label>Name</label>
+								<Field
+									name="name"
+									component="input"
+									type="text"
+									placeholder="Name"
+								/>
+							</div>
+
+							<div className="buttons">
+								<button type="submit" disabled={submitting || pristine}>
+									Submit
+								</button>
```

Comment:
not that it matters much for the prototype, but you could use the `Button` component from blueprint 
and then add the `intent={ 'primary' }` prop to increase the prominence of this button